sklearn/discriminant_analysis.py

```diff
@@ -623,11 +623,16 @@ class QuadraticDiscriminantAnalysis(BaseEstimator, ClassifierMixin):
         Discriminant Analysis
     """
 
-    def __init__(self, priors=None, reg_param=0., store_covariances=False,
-                 tol=1.0e-4):
+    def __init__(self, priors=None, reg_param=0., store_covariance=False,
+                 tol=1.0e-4, store_covariances=False):
+        if store_covariances is not None:
```

Comment:
Oh I get you now. Since `0.19` deprecates `store_covariances` I don't think we need to add this particular deprecation warning, the following change in `fit` should suffice:
```
self.store_covariance = store_covariances
```